region/lowernorfair/east.json

```diff
@@ -222,16 +222,29 @@
                   {"or": [
                     {"and": [
                       "heatProof",
-                      "canKillYellowSpacePirates"
+                      {"enemyKill":{
+                        "enemies": [
+                          [
+                            "Yellow Space Pirate (standing)"
+                          ]
+                        ],
+                        "excludedWeapons": [ "Bombs", "PowerBomb" ]
```

Comment:
This is the runway where PowerBombPeriphery would work. It allows you to kill them with 5 PBs without breaking the floor.

However, it is a situational weapon which won't be evaluated automatically. I think maybe we need another property called `includeWeapons` or something, which we can use to add situational weapons to a specific `enemyKill` without having to define a lite of `explicitWeapons`.

Reply:
I made an "or" choice between two different enemyKill where one has PowerBombPeriphery as the only explicitWeapon. Maybe that would work? Also I made a video executing the rather ridicilous strat just to get a one-time runway open: https://youtu.be/xGtUHJgZlSk